src/validation/entities/schemas/hex.schema.ts

```diff
@@ -1,4 +1,6 @@
 import { z } from 'zod';
 import { isHex } from 'viem';
 
-export const HexSchema = z.string().refine(isHex);
+export const HexSchema = z.string().refine(isHex, {
+  message: 'Invalid hex string',
+});
```

Comment:
The `isHex` requires a string to start with `0x` right? If so, I would add this to the error message as a string like `fff` can be considered an hex string for the clients.

Reply:
Good point. I updated the message to be `'Invalid "0x" notated hex string"` in 30596f4249a57df3c13f8aedd61c025d817dd81b.